skbio/sequence/_iupac_sequence.py

```diff
@@ -19,6 +19,13 @@
 from ._sequence import Sequence
 
 
+def _invert_case(array):
```

Comment:
I'm not 100% convinced this needs to be a function, but it should probably be named `_invert_ascii_case`